content/guides/userguides/accessing-metadata-via-tdr.md

```diff
@@ -0,0 +1,127 @@
+---
+date: "2021-12-25"
+description: "Overview of accessnig proejct metadata via the Terra Data Repository (TDR)."
+draft: false
+title: "Accessing Metadata via TDR"
+---
+
+# Accessing Metadata via TDR
+
+The [Terra Data Repository](https://support.terra.bio/hc/en-us/sections/4407099323675-Terra-Data-Repository)
+(TDR) offers a complete and authoritative source for the metadata made 
+available by the Data Portal data browser. This guide will walk you through the 
+process of accessing metadata stored within TDR.
+
+For detailed information regarding the structure of the data within TDR, see the
+[Terra Data Repository schema](https://github.com/HumanCellAtlas/dcp2/blob/main/docs/dcp2_system_design.rst#terra-data-repository-schema).
+
+
+## Running BigQuery SQL Queries
```

Comment:
Consider moving this paragraph down so that it comes immediately before the section on writing the queries. That way the steps are laid out in the order a user would actually follow them.